libpius/signer.py

```diff
@@ -33,10 +33,14 @@ class PiusSigner(object):
   GPG_SIG_BEG = '[GNUPG:] BEGIN_SIGNING'
   GPG_SIG_CREATED = '[GNUPG:] SIG_CREATED'
 
-  def __init__(self, signer, mode, keyring, gpg_path, tmpdir, outdir,
-               encrypt_outfiles, mail, mailer, verbose, sort_keyring,
+  def __init__(self, signer, force_signer, mode, keyring, gpg_path, tmpdir,
+               outdir, encrypt_outfiles, mail, mailer, verbose, sort_keyring,
                policy_url, mail_host):
     self.signer = signer
+    self.force_signer = force_signer
+    self.forced_signer = self.signer
```

Comment:
I missed this chunk of code here on my first read through, sorry.

OK, so I think the better thing to do here would be to do something like:

```
if force_singer:
  self.signer += '!'
```

and then just continue to use signer...